src/platform/forms/tests/forms.unit.spec.js

```diff
@@ -0,0 +1,208 @@
+import { expect } from 'chai';
```

Comment:
This is basically a file rename + adding some tests, right?

Reply:
correct